CHANGES.md

```diff
@@ -46,6 +46,10 @@ OpenSSL 4.0
 
    *Ryan Hooper*
 
+ * Reject CRLs with invalid Certificate Issuer when IDP Indirect != TRUE.
```

Comment:
Something like `Reject CRLs with Certificate Issuer extension in a certificate revocation entry when Indirect flag is not TRUE in IDP extension of the CRL.` would be better.